sys/net/gnrc/link_layer/lorawan/gnrc_lorawan.c

```diff
@@ -43,6 +43,9 @@ static inline void gnrc_lorawan_mlme_reset(gnrc_lorawan_t *mac)
     mac->mlme.pending_mlme_opts = 0;
     mac->rx_delay = (CONFIG_LORAMAC_DEFAULT_RX1_DELAY / MS_PER_SEC);
     mac->mlme.nid = CONFIG_LORAMAC_DEFAULT_NETID;
+    mac->mlme.adr = IS_ACTIVE(CONFIG_LORAMAC_DEFAULT_ADR) ? true : false;
+    mac->mlme.adr_ack = false;
```

Comment:
I think we discussed this, it made sense to have this symbols to help maintain the code later.

Reply:
I don't remember having such a discussion to be honest. But IMO it doesn't make sense, since this information is already included in the counter.

If you are really concerned about maintainability, you can always add `static inline` helpers (e.g `should_set_ack_bit(int adr_ack_cnt)`.